pkgs/development/python-modules/shamir-mnemonic/default.nix

```diff
@@ -17,6 +17,6 @@ buildPythonPackage rec {
     description = "Reference implementation of SLIP-0039";
     homepage = "https://github.com/trezor/python-shamir-mnemonic";
     license = lib.licenses.mit;
```

Comment:
You might also want to change `meta` to `meta = with lib;` (as you did with pkgs/development/python-modules/trezor/default.nix) to keep it consistent. I'd be really grateful for this change!